src/rewriter.coffee

```diff
@@ -183,6 +183,7 @@ exports.Rewriter = class Rewriter
       [prevTag] = prevToken = if i > 0 then tokens[i - 1] else []
       [nextTag] = nextToken = if i < tokens.length - 1 then tokens[i + 1] else []
       stackTop  = -> stack[stack.length - 1]
+      stackTop2 = -> stack[stack.length - 2]
```

Comment:
A name like `stackTop2` implies to me that it’s returning two elements, not that it’s referring to the second highest item in the stack. Maybe `stackSecond`?

Or maybe do without the helper function? It’s only called in one place, we could just put `stack[stack.length - 2]` there.